hydra-http-form.c

```diff
@@ -1613,6 +1613,11 @@ void usage_http_form(const char *service) {
          "You can specify a header without escaping the colons, but that way you will not\n"
          "be able to put colons in the header value itself, as they will be interpreted by\n"
          "hydra as option separators.\n"
+         "Note: to attack multiple targets, you only need to pass the path to the file containing the targets with parameters,\n"
```

Comment:
please move this text + its examples below the other examples as it looks weird if you have two following sections of "Examples:". and please mention `(-M)` in your text so people know what option you are talking about, lots of users are newbies. thanks :)